pkg/js/transformer/dsltojson.ts

```diff
@@ -88,6 +88,10 @@ class OpenFgaDslListener extends OpenFGAListener {
   };
 
   exitRelationDeclaration = (ctx: RelationDeclarationContext) => {
+    if (!ctx.relationName()) {
+      return;
+    }
+
```

Comment:
Can you also add a check on line 98? `if (!rewrites?.length) { return }`

It should not be possible to have a relation with no definition